ajax/libs/progressStep/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "progressStep",
+  "filename": "progressStep.min.js",
+  "version": "1.0.3",
+  "description": "Dynamically renders a progress bar with numbered steps.",
+  "author": {
+    "name": "Matthew Eagar",
+    "url": "https://github.com/mateagar/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mateagar/progressStep.git"
+  },
+  "keywords": [
+    "progress",
+    "bar",
+    "step",
+    "navigation"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mateagar/progressStep.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "progressStep*.+(js|map)"
```

Comment:
Except this part, everything looks good to me.
Let's use `progressStep*.+(js|map|css)`.
Because if the author add CSS file in the future, this pattern also can grab the CSS file. Also, we don't have to update the pattern of this lib frequently.
Many thanks